### PR TITLE
feat(input): Handle multiple sourceDir

### PR DESCRIPTION
**Pull request by Palapapaa**
Here is the pull request for handling multiple sourceDir.

You just have to repeat the -s parameter like this
`ts-node ./src/bin/svg-to-ts.ts -s ./inputfiles -s ./inputfilesBis`

It could also be implemented by using dots like this
`ts-node ./src/bin/svg-to-ts.ts -s ./inputfiles,./inputfilesBis` (not the way I implemented it but it's a possibilty).

I've tried to keep the changes as simple as possible.

### CODE REVIEW

File: src/bin/svg-to-ts.ts [hunk 1]
```diff
@@ -8,21 +8,32 @@ const DEFAULTS = {
...
-  .option('-s --srcDirectory <string>', 'name of the source directory', DEFAULTS.sourceDirectory)
+  .option('-s --srcDirectory <value>', 'name of the source directory', collect, [])
   .option('-o --outputDirectory <string>', 'name of the output directory', DEFAULTS.outputDirectory)
   .parse(process.argv);
 
-const { typeName, fileName, prefix, interfaceName, srcDirectory, outputDirectory } = commander;
+const { typeName, fileName, prefix, interfaceName, outputDirectory } = commander;
+
+// Because of commander adding default value to params
+// See: https://stackoverflow.com/questions/30238654/commander-js-collect-multiple-options-always-include-default
```
Comment (kreuzerk) [inline src/bin/svg-to-ts.ts]: 👍

File: src/lib/convert.ts [hunk 2]
```diff
@@ -19,21 +20,34 @@ export interface ConvertionOptions {
   prefix: string;
   fileName: string;
   interfaceName: string;
-  srcDirectory: string;
+  srcDirectory: string[];
```
Comment (kreuzerk) [inline src/lib/convert.ts]: Maybe we could rename it to srcDirecotries, what do you think?

**Comment by kreuzerk** [comment]
Cool stuff! 👍 I added a small comment. But besides that everything looks good.
In the future, I still think it makes sense to consider supporting the /** syntax.

I create a ticket for that.
Well done! Thx

File: src/lib/convert.ts (hunk 2 above)
Comment (Palapapaa) [reply src/lib/convert.ts]: If also thought so.

Maybe I can rename here, but I think we should let the CLI parameter's name as it is.

Comment (kreuzerk) [reply src/lib/convert.ts]: Yes - I also think so - let's rename it internally but keep the CLI as it is. So it makes sense if you want to pass in a single file and we are not introducing a brekaing change

Comment (Palapapaa) [reply src/lib/convert.ts]: Done =)

**Pull request merged**